capi/include/blazesym.h

```diff
@@ -49,7 +53,10 @@ typedef enum blaze_sym_type {
    * The symbol is a variable.
    */
   BLAZE_SYM_VAR,
-} blaze_sym_type;
+};
+#ifndef __cplusplus
+typedef uint8_t blaze_sym_type;
```

Comment:
it's a pretty clever abuse of C++ not requiring `enum` prefix :)

I think alternative would be to use packed enum fields (e.g., https://takashiidobe.com/gen/pack-enums-in-c), which would use enum consistently.

WDYT?

Reply:
This is `cbindgen` generated, based on `repr(u8)`. I don't believe there is an option to force this attribute, because `repr(packed)` in Rust only works on `struct` and `union`, not `enum`.